src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -536,14 +536,9 @@ bool NotationViewInputController::needSelect(const ClickContext& ctx) const
     if (!ctx.hitElement) {
         return false;
     }
-
-    bool result = true;
-
-    if (ctx.event->button() == Qt::MouseButton::RightButton) {
-        result = !viewInteraction()->selection()->range()->containsPoint(ctx.logicClickPos);
-    }
```

Comment:
This function determines what happens during mouse *press*/down - and in that case, which button you press is irrelevant (that only becomes relevant on mouse release).  But whether CTRL is held down, or the element is already selected definitely is relevant.